cli/Valet/Site.php

```diff
@@ -570,11 +570,33 @@ public function trustCertificate($crtPath, $url)
             $crtPath
         ));
 
-        $this->cli->run(sprintf(
-            'certutil -d $HOME/.mozilla/firefox/*.default -A -t TC -n "%s" -i "%s"',
-            $url,
-            $crtPath
-        ));
+        foreach ($this->getFirefoxProfilePath() as $path) {
+            $this->cli->run(sprintf(
+                'certutil -d ' . $path . ' -A -t TC -n "%s" -i "%s"',
+                $url,
+                $crtPath
+            ));
+        }
```

Comment:
```suggestion
        $paths = [
            'sql:$HOME/.pki/nssdb',
            ...$this->getFirefoxProfilePath(),
        ];
        foreach ($paths as $path) {
            $this->cli->run(sprintf(
                "certutil -d {$path} -A -t TC -n '%s' -i '%s'",
                $url,
                $crtPath,
            ));
        }
```

You could also remove the lines above your new code, merge the paths to avoid some duplication and make it easier to implement the same code for a future issue that is similar (another browser).

Reply:
Right now, I cannot test your code...  correct me if I wrong but this change is for chromiun and another web browsers

so, in this case isn't better create another pull requests? I think is better add your changes after this pull.